packages/ui/src/components/SessionTasks/tasks/TaskChooseOrganization/__tests__/TaskChooseOrganization.test.tsx

```
@@ -9,6 +11,55 @@ import {
 } from '@/ui/components/OrganizationSwitcher/__tests__/test-utils';
 
 import { TaskChooseOrganization } from '..';
+import type { FakeOrganizationParams } from '../../../../CreateOrganization/__tests__/CreateOrganization.test';
+
+type FakeOrganizationParams = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FakeOrganizationParams'. Consider to delete it or rename it.

'FakeOrganizationParams' is defined here:


(lint/suspicious/noRedeclare)